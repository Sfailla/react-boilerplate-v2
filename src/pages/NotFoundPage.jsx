import React, { Fragment } from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => {
	return (
		<Fragment>
			404! -- <Link to='/'>Go back to Home</Link>
		</Fragment>
	);
};

export default NotFoundPage;
