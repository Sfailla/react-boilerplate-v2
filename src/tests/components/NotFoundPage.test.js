import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../pages/NotFoundPage';

test('should render NotFoundPage correctly', () => {
	const wrapper = shallow(<NotFoundPage />);
	expect(wrapper).toMatchSnapshot();
});
