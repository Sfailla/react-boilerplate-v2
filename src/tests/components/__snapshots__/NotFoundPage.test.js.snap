// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<Fragment>
  404! -- 
  <Link
    replace={false}
    to="/"
  >
    Go back to Home
  </Link>
</Fragment>
`;
